projects/common/lib/components/consent-modal/consent-modal.component.ts

```diff
@@ -54,11 +54,11 @@ export class ConsentModalComponent extends AbstractHttpService implements Contro
 
     protected _headers: HttpHeaders = new HttpHeaders();
     @Input() processName: string;
-    @Input() application: any;
+    @Input() application: ApplicationBase;
```

Comment:
Two points on this one:

1. ApplicationBase, as a class, shouldn't exist in the library and instead should exist in MSP itself.  It's state management specific to that application and isn't used in other applications (PRIME, FPC, etc).

2. It looks like it's only used to update the "infoCollectionAgreement" boolean value. I would re-write this so that it doesn't actually write to the boolean itself, but instead just has an @Output() that emits when the boolean would be changed.  For example, `@Output() accept` and then in the examples `accept=onAccept($event)`.  It looks like it never sets the value to false so accept would be sufficient.